scripts/run_tests.sh

```diff
@@ -45,7 +45,7 @@ if [ "$run_docker" == 1 ]; then
         "/kb/dev_container/narrative/src/config.json.templ:/kb/dev_container/narrative/src/config.json" \
         --template \
         "/kb/dev_container/narrative/src/config.json.templ:/kb/dev_container/narrative/kbase-extension/static/kbase/config/config.json" \
-        kbase-narrative --no-browser --NotebookApp.allow_origin="*" --ip=$IP_ADDRESS --port=$JUPYTER_PORT 2>&1 | tee $OUTPUT_FILE &
```

Comment:
Using the `NotebookApp.allow_origin` and `ip` flags here was including them twice, like `allow_origin=["*", "*"]` since these are already set by the run script. I'm not sure when or how it got switched to add them together instead of override, but removing them from the test runner fixes it.